### PR TITLE
[MO] - [Parallel namespace test] -> ConnectST

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature
- Refactoring

### Description

This PR adding support for parallel namespace execution for `ConnectST` suite.
 
### Checklist

- [x] Write tests
- [x] Make sure all tests pas

### CODE REVIEW

**Comment by see-quick** [comment]
Azure regression is also green :) 
image

**Review by scholzj — APPROVED** [review]

File: systemtest/src/main/java/io/strimzi/systemtest/resources/kubernetes/NetworkPolicyResource.java [hunk 1]
```diff
@@ -157,7 +158,13 @@ public static void allowNetworkPolicySettingsForResource(ExtensionContext extens
             .addToMatchLabels(Constants.KAFKA_CLIENTS_LABEL_KEY, Constants.KAFKA_CLIENTS_LABEL_VALUE)
             .build();
 
-        if (kubeClient(resource.getMetadata().getNamespace()).listPods(labelSelector).size() == 0) {
+        final String namespaceName = StUtils.isParallelNamespaceTest(extensionContext) ?
+            // if parallel namespace test use namespace from store
+            extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get(Constants.NAMESPACE_KEY).toString() :
```
Comment (Frawless) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/kubernetes/NetworkPolicyResource.java:163]: empty line

**Review by Frawless — APPROVED** [review]

**Pull request merged**